### ISSUE TITLE
Images from Ray Gervais's post are broken

### ISSUE THREAD

**Issue by c3ho** [Seneca-CDOT/telescope#774]
**What happened**:
Images are not being rendered on [dev](dev.telescope.cdot.systems)  for Ray Gervais's post

![image](https://user-images.githubusercontent.com/18711727/75737134-0929b200-5ccd-11ea-908c-c401b85d5780.png)

**What should have happened**:
Images should have been rendered

**How to reproduce it (as precise as possible)**:
Go to the link above

**Anything else we need to know?**:
If you go to his [blog](https://raygervais.dev/article/reduced-container-sizes-with-multi-stage-docker-builds/), you will find images are actually fine and aren't broken

![image](https://user-images.githubusercontent.com/18711727/75737335-789fa180-5ccd-11ea-9e81-07ed483cc219.png)

**Environment**:

- OS: Windows 10
- Browser: Chrome

**Comment by Silvyre** [comment]
@raygervais

**Comment by raygervais** [comment]
I'm fairly confident this is on my side, since the references are relative path for my Gridsome site.

```html
<img alt="Cover image" src="/assets/static/pawel-czerwinski-f_sUgtC7lV0-unsplash.07cc2b7.dc7372eb230e70039d4d363df4472818.jpg" data-src="/assets/static/pawel-czerwinski-f_sUgtC7lV0-unsplash.07cc2b7.dc7372eb230e70039d4d363df4472818.jpg" data-srcset="/assets/static/pawel-czerwinski-f_sUgtC7lV0-unsplash.a67b0b2.dc7372eb230e70039d4d363df4472818.jpg 480w, /assets/static/pawel-czerwinski-f_sUgtC7lV0-unsplash.07cc2b7.dc7372eb230e70039d4d363df4472818.jpg 860w" data-sizes="(max-width: 860px) 100vw, 860px" class="g-image g-image--lazy g-image--loaded" srcset="/assets/static/pawel-czerwinski-f_sUgtC7lV0-unsplash.a67b0b2.dc7372eb230e70039d4d363df4472818.jpg 480w, /assets/static/pawel-czerwinski-f_sUgtC7lV0-unsplash.07cc2b7.dc7372eb230e70039d4d363df4472818.jpg 860w" sizes="(max-width: 860px) 100vw, 860px" width="860"><noscript><img src="/assets/static/pawel-czerwinski-f_sUgtC7lV0-unsplash.07cc2b7.dc7372eb230e70039d4d363df4472818.jpg" class="g-image g-image--loaded" width="860" alt="Cover image"></noscript>
```

I will try using full path including URL to see if that fixes it on my end.

**Comment by raygervais** [comment]
@Silvyre, I've opened up an issue with Gridsome (Vue alt of Gatsby essentially), unsure if they will fix this in time for 0.8 so I'd remove from the release?

**Comment by Silvyre** [comment]
Ah, sure, good call; let's do that
